packages/sdks/react-sdk/src/components/ApplicationsPortal.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {
+  lazy,
+  Suspense,
+  useEffect,
+  useImperativeHandle,
+  useState,
+} from 'react';
+import Context from '../hooks/Context';
+import { ApplicationsPortalProps } from '../types';
+
+// web-component code uses browser API, but can be used in SSR apps, hence the lazy loading
+const ApplicationsPortalWC = lazy(async () => {
+  await import('@descope/applications-portal-widget');
+
+  return {
+    default: ({
+      projectId,
+      baseUrl,
+      baseStaticUrl,
+      innerRef,
+      widgetId,
+      theme,
+      debug,
+    }) => (
+	<descope-applications-portal-widget
+        project-id={projectId}
+        widget-id={widgetId}
+        base-url={baseUrl}
+        base-static-url={baseStaticUrl}
+        theme={theme}
+        debug={debug}
+        ref={innerRef}
+      />
+    ),
+  };
+});
+
+const ApplicationsPortal = React.forwardRef<
+  HTMLElement,
+  ApplicationsPortalProps
+>(({ logger, theme, debug, widgetId, onLogout }, ref) => {
+  const [innerRef, setInnerRef] = useState(null);
+
+  useImperativeHandle(ref, () => innerRef);
+
+  const { projectId, baseUrl, baseStaticUrl } = React.useContext(Context);
+
+  useEffect(() => {
+    if (innerRef && logger) {
+      innerRef.logger = logger;
+    }
+  }, [innerRef, logger]);
+
+  useEffect(() => {
+    if (innerRef && onLogout) {
+      innerRef.addEventListener('logout', onLogout);
```

Comment:
is this a part of the applications portal?

Reply:
Mistake